main/DDF/EscherSimpleProperty.cs

```diff
@@ -31,7 +31,7 @@ namespace NPOI.DDF
     /// </summary>
     public class EscherSimpleProperty : EscherProperty
     {
-        protected int propertyValue;
```

Comment:
Do we really need to introduce a breaking change?

Reply:
This change will not introduce breaking change. The `EscherSimpleProperty` class use `PropertyValue` property to expose this field and all subclass of  `EscherSimpleProperty`  use `PropertyValue` property instead of  this field.